source/class/cv/plugins/diagram/AbstractDiagram.js

```diff
@@ -285,7 +298,7 @@ qx.Class.define('cv.plugins.diagram.AbstractDiagram', {
             this.cache[ key ].xhr.dispose();
           }
           var xhr = new qx.io.request.Xhr(url);
-          var self = this;
```

Comment:
During compile I get
```
cv.plugins.diagram.AbstractDiagram: [305,50] warning: Unresolved use of symbol self._onSuccessiagram
cv.plugins.diagram.AbstractDiagram: [306,54] warning: Unresolved use of symbol self._onStatusError
```
Reinserting `var self = this;` fixed that

Reply:
We can use this here, because the context is provided to the listeners.